### DISCUSSION TITLE
Does CLEAR CACHE options effect current cube or all cubes?

### DISCUSSION THREAD

**Discussion by TinyTimothy2** [TabularEditor/TabularEditor3#1395] [Data Modeling]
Does CLEAR CACHE options effect current cube or all cubes?
Ie options in dax studio.
It seems dangerous to clear cache testing code in single model if it can affect all SSAS models.

**Comment by otykier** [comment]
I *think* that cache clearing is at the database (model) level, but I'm not 100% sure. At least, the clear cache XMLA request must [specify the database](https://learn.microsoft.com/en-us/analysis-services/instances/clear-the-analysis-services-caches?view=asallproducts-allversions#step-2-run-the-query) so intuitively it seems that it would only affect that database. I was not able to find any information to the contrary, but you should ask Microsoft to be sure.
